ts/sagas/profile.ts

```diff
@@ -115,6 +115,11 @@ function* createOrUpdateProfileSaga(
     return;
   }
 
+  if (response.value.status === 429) {
```

Comment:
why logging out the user?

Reply:
what is the expected behaviour in this case? 

Like this:

```suggestion
  if (response.value.status === 429) {
      const error: Error = Error(
        response.value.value.title || I18n.t("profile.errors.toomanyrequest")
      );
  }
```

Reply:
errors are already covered by the next block `if (response.value.status !== 200)`

Reply:
Fix in a641da6